src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(int firstSide, int secondSide, Color color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        super.color = color;

```

Comment:
The assignment of `color` should be done through the constructor of the superclass `Figure`. Use `super(color)` to set the color.